src/tests/test_elements.cc

```diff
@@ -0,0 +1,84 @@
+
+#include <cassert>
+#include <exception>
+#include <iostream>
+#include <string>
+#include <votca/tools/elements.h>
+
+using namespace std;
+using namespace votca::tools;
+
+int main(void) {
```

Comment:
Maybe this is a good point to use something like boost test framework?

Reply:
Hmm I will have to look into it something else I don't know about. But more unit testing and pull requests will be coming your way so it's probably worth it.